OpenRA.Mods.Common/Scripting/Global/MapGlobal.cs

```diff
@@ -107,8 +107,8 @@ public string TerrainType(CPos cell)
 			return Context.World.Map.GetTerrainInfo(cell).Type;
 		}
 
-		[Desc("Returns true if there is only one human player.")]
-		public bool IsSinglePlayer { get { return Context.World.LobbyInfo.IsSinglePlayer; } }
+		[Desc("Returns true if there is only one human client.")]
+		public bool IsSinglePlayer { get { return Context.World.LobbyInfo.NonBotClients.Count() == 1; } }
```

Comment:
It actually is: https://github.com/OpenRA/OpenRA/pull/13271#discussion_r117537334. I think it is a bug to count bot-only matches as "multiplayer" matches. So either `NonBotClients` or `NonBotPlayers < 2`.